### DISCUSSION TITLE
YOURLS links not opening

### DISCUSSION THREAD

**Discussion by CyberDudeJ** [YOURLS/YOURLS#2867] [Q&A] [answered]
This may be a cause of just me needing to reinstall it but i have no idea why the pages wont load using yourls
also how to i customise it!

**Comment by CyberDudeJ** [comment]
also how do i customise the loading screen

**Comment by dgw** [comment]
Please follow the [troubleshooting guide](https://github.com/YOURLS/YOURLS/wiki/Troubleshooting-first-steps).

If you still have problems, we will need to know your YOURLS version, server OS version, PHP version, and HTTP server version—at minimum. Logs wouldn't hurt, either.

**Comment by minecraftchest1** [comment, marked as answer]
I do not understand your question.
